lib/index.js

```diff
@@ -1,16 +1,58 @@
 var Board = require("firmata");
+var bufferpack = require("bufferpack");
 
+// Constants that define the Circuit Playground Firmata command values.
 var CP_COMMAND = 0x40;
 var CP_PIXEL_SET = 0x10;
 var CP_PIXEL_SHOW = 0x11;
 var CP_PIXEL_CLEAR = 0x12;
-var CP_TONE        = 0x20;
-var CP_NO_TONE     = 0x21;
+var CP_TONE = 0x20;
+var CP_NO_TONE = 0x21;
+var CP_ACCEL_READ = 0x30;
+var CP_ACCEL_TAP = 0x31;
+var CP_ACCEL_ON = 0x32;
+var CP_ACCEL_OFF = 0x33;
+var CP_ACCEL_TAP_ON = 0x34;
+var CP_ACCEL_TAP_OFF = 0x35;
+var CP_ACCEL_READ_REPLY = 0x36;
+var CP_ACCEL_TAP_REPLY = 0x37;
+var CP_ACCEL_TAP_STREAM_ON = 0x38;
+var CP_ACCEL_TAP_STREAM_OFF = 0x39;
+var CP_ACCEL_STREAM_ON = 0x3A;
+var CP_ACCEL_STREAM_OFF = 0x3B;
+var CP_ACCEL_RANGE = 0x3C;
+var CP_CAP_READ = 0x40;
+var CP_CAP_ON = 0x41;
+var CP_CAP_OFF = 0x42;
+var CP_CAP_REPLY = 0x43;
+
+// Accelerometer constants to be passed to set_accel_range.
+var ACCEL_2G = 0;
+var ACCEL_4G = 1;
+var ACCEL_8G = 2;
+var ACCEL_16G = 3;
+
+// Constants for some of the board peripherals
+var THERM_PIN = 0;  // Analog input connected to the thermistor.
+var THERM_SERIES_OHMS = 10000.0;  // Resistor value in series with thermistor.
+var THERM_NOMINAL_OHMS = 10000.0;  // Thermistor resistance at 25 degrees C.
+var THERM_NOMIMAL_C = 25.0;  // Thermistor temperature at nominal resistance.
+var THERM_BETA = 3950.0;  // Thermistor beta coefficient.
+var CAP_THRESHOLD = 300;  // Threshold for considering a cap touch input pressed.
+// If the cap touch value is above this value it is considered touched.
 
 function Playground(options) {
   Board.call(this, options.port);
+  this.on("ready", function () {
+    this.sysexResponse(CP_COMMAND, function (data) {
+      var command = data[0] & 0x7F;
+      replyHandlers[command](data);
```

Comment:
Instead of making every operation parse all the 7-bit bytes, just do that once here.

```js
replyHandlers[command](Board.decode(data.slice(2)));
```